### PR TITLE
CI: Allow true isolation

### PR DESCRIPTION
**Pull request by mfe**
## Purpose of this PR

Added the option to test the isolation + 2 missing dependencies.
2019.4 and 2020.2 are still failing because of Timeline dependencies not yet adapted to true isolation.
Will ask what is the timeframe to fix previous versions.

We could also update our timeline dependency (trunk and 2021.1 are using 1.5.1-pre.3).
I double checked, CI is back to green when doing so.

While I was there I updated the Unity test versions.

**Complexity:**
Minimal 

**Halo Effect:**
Minimal

### CODE REVIEW

File: .yamato/upm-ci.yml
Comment (clusty) [inline .yamato/upm-ci.yml:5]: would also add trunk

Comment (clusty) [inline .yamato/upm-ci.yml:5]: would also add trunk

File: package/com.unity.formats.usd/package.json
Comment (clusty) [inline package/com.unity.formats.usd/package.json:5]: for my curiosity, what part of physics is USD using ?

**Review by clusty — APPROVED** [review]

File: .yamato/upm-ci.yml
Comment (mfe) [reply .yamato/upm-ci.yml:5]: There is already trunk

File: package/com.unity.formats.usd/package.json
Comment (mfe) [reply package/com.unity.formats.usd/package.json:5]: RigidBody

**Comment by mfe** [comment]
Timeline dependencies are expected to be fixed for < 2021 soon.
So waiting for this to happen before merging so our CI remains green.

**Review by judubu — APPROVED** [review]

**Pull request merged**
